src/components/search-form/__tests__/SearchForm.test.js

```diff
@@ -61,10 +60,30 @@ describe('components/search-form/SearchForm', () => {
                 ],
             },
         };
-        const wrapper = mount(<SearchForm onSubmit={onSubmitSpy} placeholder="search" value="cheese" />);
-        const form = wrapper.find('form');
-        form.simulate('submit', event);
-        sinon.assert.calledWithMatch(onSubmitSpy, 'cheese', event);
+        const onSubmitMock = jest.fn();
+
+        afterEach(() => {
+            onSubmitMock.mockReset();
+        });
+
+        test('should call onsubmit on form submit event', () => {
+            const wrapper = mount(<SearchForm onSubmit={onSubmitMock} placeholder="search" value="cheese" />);
+            const form = wrapper.find('form');
+
+            form.simulate('submit', event);
+            expect(onSubmitMock).toHaveBeenCalledTimes(1);
```

Comment:
Could we update this and the test below to `toBeCalledTimes`?  https://github.com/box/box-ui-elements/pull/2999#discussion_r951632495
```suggestion
            expect(onSubmitMock).toBeCalledTimes(1);
```